### ISSUE TITLE
informal treatment (enhancement)

### ISSUE THREAD

**Issue by Wjxfi** [kolserdav/ana#100]
slang, abbreviations, shortening two words into one, like "y'all" and more.

**Comment by kolserdav** [comment]
At the moment, it is undesirable to prescribe this in the application, since it would be better if it was already out of the box from ctranslate2 and argo-strain.
We only use them:  

https://github.com/kolserdav/ana/blob/3b03d696ffa99b2035a105b04805e0d5418a9ef0/packages/translate2/core/translate.py#L1-L125

**Comment by kolserdav** [comment]
https://github.com/PJ-Finlay , do you have any one?

**Comment by kolserdav** [comment]
@Wjxfi , contact me on email.

**Comment by PJ-Finlay** [comment]
> At the moment, it is undesirable to prescribe this in the application, since it would be better if it was already out of the box from [ctranslate2](https://github.com/OpenNMT/CTranslate2) and [argos-train](https://github.com/argosopentech/argos-train). We only use them:
> 
> https://github.com/kolserdav/ana/blob/3b03d696ffa99b2035a105b04805e0d5418a9ef0/packages/translate2/core/translate.py#L1-L125

I agree we would want to add examples of translating this correctly to the training data.
